styles/utils.css

```diff
@@ -0,0 +1,105 @@
+/* own utilities */
+
+.rounded-buttons {
+    display: inline-block;
+}
+
+.rounded-button {
+    background: rgb(187, 225, 215);
+    width: 9px;
+    height: 9px;
+    display: inline-block;
+    border-radius: 50%;
+    margin: 0 4px;
+}
+
+@media (min-width: 577px) {
+    .rounded-button-mobile {
+        display: none;
+    }
+    .rounded-button {
+        width: 15px;
+        height: 15px;
+    }
+}
+
+.rounded-button--active {
+    background: rgb(124, 195, 175);
+}
+
+.close {
+    display: none;
+}
+
+.active-element {
+    visibility: visible;
+    opacity: 1;
+}
+
+/* etline icons */
+
+.icon:before {
+    font-family: 'et-line';
+    content: attr(data-icon);
+    speak: none;
+    font-weight: normal;
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    display: inline-block;
+}
+
+/* social media icons */
+
+@font-face {
+    font-family: 'icomoon';
+    src: url('/resources/fonts/extra-icons-font/icomoon.eot?glned1');
+    src: url('/resources/fonts/extra-icons-font/icomoon.eot?glned1#iefix') format('embedded-opentype'), url('/resources/fonts/extra-icons-font/icomoon.ttf?glned1') format('truetype'), url('/resources/fonts/extra-icons-font/icomoon.woff?glned1') format('woff'), url('/resources/fonts/extra-icons-font/icomoon.svg?glned1#icomoon') format('svg');
+    font-weight: normal;
+    font-style: normal;
+    font-display: block;
+}
+
+.social-media-icon, .icon-link {
+    /* use !important to prevent issues with browser extensions that change fonts */
+    font-family: 'icomoon' !important;
```

Comment:
remove the important, use specificity instead

Reply:
Well, this was in the default example of the icomoon, but I think I could change it without problems